parkings/api/operator/operator_permission.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import permissions
```

Comment:
Not related to this line, but since GitHub doesn't allow commenting the commit messages putting the comment here:
> Refactor OperatorApi permission
>
> Refactor OpertorAPI has_permission as this will be shared among
> other Operator endpoints too.

This commit is just moving the code, so the title would be more descriptive as "Move OperatorApiHasPermission to separate module" (or something similar). Just "Refactor something" doesn't really tell what kind of a refactoring it is (could be e.g. move, rename, split to pieces, combine from pieces, complete rewrite, reordering methods, etc.).

Reply:
Sorry, misread the code first. It wasn't just moving the class, but actually splitting out a base class from it. So a better title would be e.g. "Operator API: Split out common base class for permissions" or similar.